emds: emds doc and emds usage in light_ctrl sample improvement (#23969)
nrfconnect/sdk-nrf: doc/nrf/libraries/others/emds.rst

Calling the :c:func:`emds_store_time_get` function in the sample automatically computes the result of the formula and returns 25360.

Data storing context
====================

Despite that the :c:func:`emds_store` function blocks interrupts to prevent context pre-emption, it is recommended to call it from a context with the highest priority to minimize the chance of context pre-emption before the function starts executing.
The EMDS “data storing finished” callback is invoked from the same context as :c:func:`emds_store`.

> If a radio is present, it is required to uninitialize the :ref:`SoftDevice Controller <nrfxlib:softdevice_controller>` and the :ref:`Multiprotocol Service Layer <mpsl_lib>` before calling :c:func:`emds_store` to ensure that no radio activity is in progress.

Reviewer: ```suggestion
If a radio is present, the :ref:`SoftDevice Controller <nrfxlib:softdevice_controller>` and the :ref:`Multiprotocol Service Layer <mpsl_lib>` must be uninitialized before calling :c:func:`emds_store` to ensure that no radio activity is in progress.
```

> If the radio exists, it is recommended to uninitialize the :ref:`SoftDevice Controller <nrfxlib:softdevice_controller>` and the :ref:`Multiprotocol Service Layer <mpsl_lib>` before calling the storing functionality to stop any potential radio activities.

Reviewer 2: If you want to keep `high priority` above, not `highest`, then clarify here that in this case (when radio is present), `emds_store` must be called from the highest priority.

I think it is good to mention that stopping radio saves power.
